include/faabric/util/queue.h

```diff
@@ -30,7 +30,7 @@ class Queue
     {
         UniqueLock lock(mx);
 
-        mq.push(value);
+        mq.emplace(std::move(value));
```

Comment:
AFAICT the new line will do the same as the first one in the worst case, but can be optimized if we pass it a certain type of value. This is necessary to have a queue of `unique_ptr`s.

Reply:
Should we have an associated `std::move` around `doDequeue` in the `dequeue` method?